layouts/fastly/helix.vcl

```diff
@@ -1879,6 +1879,11 @@ sub vcl_fetch {
   # Vary on x-ow-version-lock, to avoid caching of different versions
   set beresp.http.Vary:X-OW-Version-Lock = "";
 
+  # Vary on X-Github-Token, to avoid caching secret content
+  if (req.backend == F_AdobeRuntime || req.backend == F_UniversalRuntime || req.backend == F_GitHub) {
+    set beresp.http.Vary:X-Github-Token = "";
```

Comment:
we don't need this, since the github token would only be used for request to the content proxy.

Reply:
But you would be able to make a request to an `.md` file that has been delivered before and get the cached version, without any authentication. In fact, you could poison the cache by requesting the URL with a missing or wrong token and the 404 would be cached.

Reply:
but this feature is not about authentication in .page, but how to access private content. if .page automatically provides the token for the content, the cache is always correct.

as I said: IMO a x-github-token passed to .page should not have any effect.